sql_model/main.py

```
@@ -0,0 +1,135 @@
+        # It can be done in a single statement
+        # heroes = session.exec(select(Hero)).all()
+
+
+def select_where_heroes():
+    with Session(engine) as session:
+        statement = select(Hero).where(Hero.name == "Deadpond")
+        results = session.exec(statement)
+        for hero in results:
+            print(hero)
+        # ---------------- AND operations --------------------
+        s1 = select(Hero).where(Hero.age > 35, Hero.age < 45)
```

Ruff (0.8.2)

Local variable `s1` is assigned to but never used

Remove assignment to unused variable `s1`

(F841)